lib/response-parsers/community-entry.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+// node core modules
+
+// 3rd party modules
+const _ = require('lodash');
+const { parseXMLNode } = require('oniyi-utils-xml');
+
+// internal modules
+const xpath = require('../xpath-select');
+const { toDate, parseUserInfo } = require('./utils');
+
+const linkRelToNameMap = {
+  self: 'self',
+  edit: 'edit',
+};
+
+const parseLinks = nodes =>
+  _.reduce(
+    nodes,
+    (result, node) => {
+      const link = parseXMLNode(
+        node,
+        {
+          rel: 'string(@rel)',
+          href: 'string(@href)',
+        },
+        xpath
+      );
+      const { [link.rel || '']: name } = linkRelToNameMap;
+      /* beautify preserve:start */
+      return Object.assign(result, { [name]: link });
+      /* beautify preserve:end */
+    },
+    {}
+  );
+
+const parseMemberCount = node =>
```

Comment:
you can do parseInt as well as `firstChild.data` directly in xpath.
check [this](https://github.com/benkroeger/ibm-connections-blogs/blob/master/lib/response-parsers/blog-post.js#L43)